lib/github_service.rb

```diff
@@ -89,6 +89,20 @@ def milestones(fq_name)
       milestones_cache[fq_name] ||= Hash[service.list_milestones(fq_name, :state => :all).map { |m| [m.title, m.number] }]
     end
 
+    # Cache repositories for an hour, and fetch them if a list they doesn't
```

Comment:
sorry, this is a stupid nit but ```fetch them if a list they doesn't exist for a given user```

Reply:
Fixed by removing 😉 

But yes... I suck at the English language...